packages/client/screens/AuthenticationSignUpForm/index.tsx

```diff
@@ -479,6 +479,16 @@ const AuthenticationSignUpFormScreen: React.FC<SignUpFormScreenProps> = ({
                                         {errors.password.message}
                                     </Text>
                                 )}
+
+                                <View sx={{ marginTop: 'm' }}>
+                                    <Text
+                                        sx={{
+                                            color: 'greyLighter',
+                                        }}
+                                    >
+                                        Password must be 8 or more in length.
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** great